showcase/scripts/showcase_utilities.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+function cleanup {
+  script_dir=$1
+  cd "${script_dir}"
+  rm -rdf gapic-generator-java* google schema protobuf-* protoc-gen-grpc-java* showcase-output out
```

Comment:
Can we keep all of these folders in a temporary directory because when the script fails midway and cleanup is skipped, it's petty messy? It would be easier if there was a subdirectory under `showcase` like `build` that contained all of this.

Reply:
+1. Similar to the temp bazel folders like `bazel-out` when we were using bazel.